### PR TITLE
str #16652 Convert quickstart to java

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/TwitterStreamQuickstartDocTest.java [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+    final KeyedSink<Integer, Future<Integer>> sumSink = 
+      Sink.<Integer, Integer>fold(0, (acc, elem) -> acc + elem);
+
+    final RunnableFlow counter = tweets.map(t -> 1).to(sumSink);
+    final MaterializedMap map = counter.run(mat);
+
+    final Future<Integer> sum = map.get(sumSink);
+
+    sum.foreach(new Foreach<Integer>() {
+      public void each(Integer c) {
```
Comment (drewhk) [inline akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/TwitterStreamQuickstartDocTest.java]: lambda?

**Comment by drewhk** [comment]
LGTM

File: akka-docs-dev/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala [hunk 2]
```diff
@@ -37,19 +34,21 @@ object TwitterStreamQuickstartDocSpec {
...
+    Tweet(Author("rolandkuhn"), System.currentTimeMillis, "#akka rocks!") ::
+      Tweet(Author("patriknw"), System.currentTimeMillis, "#akka !") ::
+      Tweet(Author("bantonsson"), System.currentTimeMillis, "#akka !") ::
+      Tweet(Author("drewhk"), System.currentTimeMillis, "#akka !") ::
+      Tweet(Author("ktosopl"), System.currentTimeMillis, "#akka on the rocks!") ::
+      Tweet(Author("mmartynas"), System.currentTimeMillis, "wow #akka !") ::
+      Tweet(Author("akkateam"), System.currentTimeMillis, "#akka rocks!") ::
+      Tweet(Author("bananaman"), System.currentTimeMillis, "#bananas rock!") ::
+      Tweet(Author("appleman"), System.currentTimeMillis, "#apples rock!") ::
+      Tweet(Author("drama"), System.currentTimeMillis, "we compared #apples to #oranges!") ::
```
Comment (ktoso) [inline akka-docs-dev/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala]: +1

**Comment by ktoso** [comment]
LGTM after that foreach is lambdafied IMO

File: akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/TwitterStreamQuickstartDocTest.java (hunk 1 above)
Comment (patriknw) [inline akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/TwitterStreamQuickstartDocTest.java]: Is that possible? This is a Future.

**Pull request merged**